specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/StaticSites.json

```diff
@@ -2685,6 +2685,607 @@
         },
         "x-ms-long-running-operation": true
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/staticSites/{name}/linkedBackends/{linkedBackendName}/validate": {
+      "post": {
+        "tags": [
+          "StaticSites"
+        ],
+        "summary": "Validates that a backend can be linked to a static site",
+        "operationId": "StaticSites_ValidateBackend",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the static site",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "linkedBackendName",
+            "in": "path",
+            "description": "Name of the linked backend that should be retrieved",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "staticSiteLinkedBackendEnvelope",
+            "in": "body",
+            "description": "A JSON representation of the linked backend request properties",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/StaticSiteLinkedBackendARMResource"
+            }
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "No content"
+          },
+          "200": {
```

Comment:
>200 [](http://example.com/codeflow?start=11&length=3)

Should be 202 for async response. Ref: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#call-action-post-asynchronously](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#call-action-post-asynchronously)